packages/core/__test__/utils/elementReady.test.js

```diff
@@ -55,8 +55,8 @@ describe('TestReady', function () {
 
     const { errorMessage, errorCount } = getErrors();
 
-    expect(errorMessage).to.equal('test');
-    expect(errorCount).to.equal(1);
+    expect(errorCount).to.equal(1, 'Error not thrown');
+    expect(errorMessage).to.equal('test', 'duplication error not thrown in dev environment');
```

Comment:
Forgot to update message for test failure?

Reply:
Fixed ;)